2024-04-30 14:15:21,020 INFO    MainThread:116757 [wandb_setup.py:_flush():76] Current SDK version is 0.16.6
2024-04-30 14:15:21,020 INFO    MainThread:116757 [wandb_setup.py:_flush():76] Configure stats pid to 116757
2024-04-30 14:15:21,020 INFO    MainThread:116757 [wandb_setup.py:_flush():76] Loading settings from /home/hghallab/.config/wandb/settings
2024-04-30 14:15:21,020 INFO    MainThread:116757 [wandb_setup.py:_flush():76] Loading settings from /home/hghallab/Dvlp/GANs/DCGAN/wandb/settings
2024-04-30 14:15:21,020 INFO    MainThread:116757 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-04-30 14:15:21,020 INFO    MainThread:116757 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-04-30 14:15:21,020 INFO    MainThread:116757 [wandb_setup.py:_flush():76] Applying login settings: {'api_key': '***REDACTED***'}
2024-04-30 14:15:21,020 INFO    MainThread:116757 [wandb_setup.py:_flush():76] Applying login settings: {'api_key': '***REDACTED***'}
2024-04-30 14:15:21,020 INFO    MainThread:116757 [wandb_setup.py:_flush():76] Applying login settings: {}
2024-04-30 14:15:21,020 INFO    MainThread:116757 [wandb_init.py:_log_setup():521] Logging user logs to /home/hghallab/Dvlp/GANs/DCGAN/wandb/run-20240430_141521-78izv9ec/logs/debug.log
2024-04-30 14:15:21,020 INFO    MainThread:116757 [wandb_init.py:_log_setup():522] Logging internal logs to /home/hghallab/Dvlp/GANs/DCGAN/wandb/run-20240430_141521-78izv9ec/logs/debug-internal.log
2024-04-30 14:15:21,021 INFO    MainThread:116757 [wandb_init.py:_jupyter_setup():467] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7be527a5e140>
2024-04-30 14:15:21,023 INFO    MainThread:116757 [wandb_init.py:init():561] calling init triggers
2024-04-30 14:15:21,023 INFO    MainThread:116757 [wandb_init.py:init():568] wandb.init called with sweep_config: {}
config: {'seed': 12, 'debug': False, 'model_name': 'DCGANsmall|templates', 'loss_function': 'BCELoss', 'train_bs': 8, 'valid_bs': 8, 'img_size': [64, 64], 'epochs': 1000, 'lr': 0.0002, 'n_accumulate': 4, 'device': device(type='cpu')}
2024-04-30 14:15:21,023 INFO    MainThread:116757 [wandb_init.py:init():611] starting backend
2024-04-30 14:15:21,023 INFO    MainThread:116757 [wandb_init.py:init():615] setting up manager
2024-04-30 14:15:21,025 INFO    MainThread:116757 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2024-04-30 14:15:21,026 INFO    MainThread:116757 [wandb_init.py:init():623] backend started and connected
2024-04-30 14:15:21,030 INFO    MainThread:116757 [wandb_run.py:_label_probe_notebook():1299] probe notebook
2024-04-30 14:15:21,030 INFO    MainThread:116757 [wandb_run.py:_label_probe_notebook():1309] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2024-04-30 14:15:21,030 INFO    MainThread:116757 [wandb_init.py:init():715] updated telemetry
2024-04-30 14:15:21,036 INFO    MainThread:116757 [wandb_init.py:init():748] communicating run to backend with 90.0 second timeout
2024-04-30 14:15:21,674 INFO    MainThread:116757 [wandb_run.py:_on_init():2357] communicating current version
2024-04-30 14:15:22,026 INFO    MainThread:116757 [wandb_run.py:_on_init():2366] got version response 
2024-04-30 14:15:22,026 INFO    MainThread:116757 [wandb_init.py:init():799] starting run threads in backend
2024-04-30 14:15:22,202 INFO    MainThread:116757 [wandb_run.py:_console_start():2335] atexit reg
2024-04-30 14:15:22,202 INFO    MainThread:116757 [wandb_run.py:_redirect():2190] redirect: wrap_raw
2024-04-30 14:15:22,203 INFO    MainThread:116757 [wandb_run.py:_redirect():2255] Wrapping output streams.
2024-04-30 14:15:22,203 INFO    MainThread:116757 [wandb_run.py:_redirect():2280] Redirects installed.
2024-04-30 14:15:22,203 INFO    MainThread:116757 [wandb_init.py:init():842] run started, returning control to user process
2024-04-30 14:15:22,203 INFO    MainThread:116757 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-04-30 14:15:22,203 INFO    MainThread:116757 [wandb_init.py:_pause_backend():432] pausing backend
2024-04-30 14:15:22,210 INFO    MainThread:116757 [wandb_init.py:_resume_backend():437] resuming backend
2024-04-30 14:15:22,211 INFO    MainThread:116757 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-04-30 14:15:22,211 INFO    MainThread:116757 [wandb_init.py:_pause_backend():432] pausing backend
2024-04-30 14:15:22,216 INFO    MainThread:116757 [wandb_init.py:_resume_backend():437] resuming backend
2024-04-30 14:15:22,283 INFO    MainThread:116757 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-04-30 14:15:22,283 INFO    MainThread:116757 [wandb_init.py:_pause_backend():432] pausing backend
2024-04-30 14:15:22,289 INFO    MainThread:116757 [wandb_init.py:_resume_backend():437] resuming backend
2024-04-30 14:15:22,291 INFO    MainThread:116757 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-04-30 14:15:22,291 INFO    MainThread:116757 [wandb_init.py:_pause_backend():432] pausing backend
2024-04-30 14:15:22,304 INFO    MainThread:116757 [wandb_init.py:_resume_backend():437] resuming backend
2024-04-30 14:15:22,309 INFO    MainThread:116757 [wandb_watch.py:watch():51] Watching
2024-04-30 14:15:22,309 INFO    MainThread:116757 [wandb_watch.py:watch():51] Watching
2024-04-30 14:19:08,652 INFO    MainThread:116757 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-04-30 14:19:08,659 INFO    MainThread:116757 [wandb_init.py:_pause_backend():432] pausing backend
2024-04-30 14:19:08,668 INFO    MainThread:116757 [wandb_init.py:_resume_backend():437] resuming backend
2024-04-30 14:19:08,800 INFO    MainThread:116757 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-04-30 14:19:08,800 INFO    MainThread:116757 [wandb_init.py:_pause_backend():432] pausing backend
2024-04-30 14:19:08,807 INFO    MainThread:116757 [wandb_init.py:_resume_backend():437] resuming backend
2024-04-30 14:19:08,820 INFO    MainThread:116757 [wandb_run.py:_finish():2064] finishing run houda222/DCGAN/78izv9ec
2024-04-30 14:19:08,820 ERROR   MainThread:116757 [jupyter.py:save_history():437] Run pip install nbformat to save notebook history
2024-04-30 14:19:08,820 INFO    MainThread:116757 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-04-30 14:19:08,820 INFO    MainThread:116757 [wandb_init.py:_jupyter_teardown():449] cleaning up jupyter logic
2024-04-30 14:19:08,820 INFO    MainThread:116757 [wandb_run.py:_atexit_cleanup():2304] got exitcode: 0
2024-04-30 14:19:08,820 INFO    MainThread:116757 [wandb_run.py:_restore():2287] restore
2024-04-30 14:19:08,820 INFO    MainThread:116757 [wandb_run.py:_restore():2293] restore done
